### PR TITLE
add flashcards

### PR DESCRIPTION
**Pull request by Froggerson**
I have flashcards now

### CODE REVIEW

**Comment by ESaracay** [comment]
This code is really clean and makes a lot of sense.

**Review by ESaracay — APPROVED** [review]

**Comment by Froggerson** [comment]
Merging Flashcards to setup

File: frontend/src/App.test.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
-import React from 'react';
-import { render } from '@testing-library/react';
-import App from './App';
+/*import React from 'react';
```
Comment (tylerhou) [inline frontend/src/App.test.js:1]: Delete unused code; we can always find it again via source control.

File: frontend/src/flashcards/Flashcard.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+/*
+  This is the styling for the main flashcard.
+*/
+const flipStyle ={
+  width:"300px",
+  height:"350px",
+  margin:"10px",
+};
+
+/*
```
Comment (tylerhou) [inline frontend/src/flashcards/Flashcard.js:15]: Good comments.

File: frontend/src/flashcards/Flashcard.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  text, translation, and image.
+  It renders a flashcard with both a front and a back.
+  It also uses the react-card-flip library, so make 
+  sure to install it if possible!
+*/
+class Flashcard extends React.Component {
+    constructor(props) {
+      super(props);
+      this.handleClick=this.handleClick.bind(this);
+      this.state = { isFlipped: false};
```
Comment (tylerhou) [inline frontend/src/flashcards/Flashcard.js:26]: Please format.

File: frontend/src/flashcards/FlashcardBack.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+/*
+  Styling for the back of the flashcard.
+*/
+const backStyle={
```
Comment (tylerhou) [inline frontend/src/flashcards/FlashcardBack.js:6]: Migrate to a CSS file now that Evan has included CSS support?

File: frontend/src/flashcards/FlashcardContainer.js [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+    display:"flex",
+    alignItems: "center",
+    justifyContent: "space-around",
+    flexFlow: "row wrap",
+}
+/*
+    Holds a container of flashcards.
+    TODO: Integrate backend to retrieve flashcard data
+*/
+class FlashcardContainer extends React.Component{
```
Comment (tylerhou) [inline frontend/src/flashcards/FlashcardContainer.js:41]: Format

File: frontend/src/flashcards/FlashcardFront.js [hunk 6]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+/* Styling for the front of the flashcard*/
+const frontStyle={
```
Comment (tylerhou) [inline frontend/src/flashcards/FlashcardFront.js:4]: Format

File: frontend/src/homePage.js [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+      <NavBar />
+      <LandingPage />
+      <About />
+    </div>
+  );
+}
+
+ReactDOM.render(
+    <Home></Home>,
+  document.getElementById("root")
```
Comment (tylerhou) [inline frontend/src/homePage.js:19]: Format

File: frontend/src/main-components/Sidebar.js [hunk 8]
```diff
@@ -35,12 +35,12 @@ const Sidebar = (props) => {
   return (
     <Side id="side" class="flex-container">
       
-      <SideIcon link="/" icon={homeIcon}></SideIcon>
+      <SideIcon link="/myFolders" icon={homeIcon}></SideIcon>
```
Comment (tylerhou) [inline frontend/src/main-components/Sidebar.js:38]: Make path names consistent.

File: src/main/java/com/google/sps/servlets/ReactServlet.java [hunk 9]
```diff
@@ -14,7 +14,7 @@
   //If User does not exist in database, we create a new User entity. Page redirected to the /MyFolder Url
```
Comment (tylerhou) [inline src/main/java/com/google/sps/servlets/ReactServlet.java:14]: `HomePageServlet`?

Thanks for renaming the React bundles.

File: src/main/java/com/google/sps/servlets/ReactServlet2.java [hunk 10]
```diff
@@ -8,13 +8,13 @@
 import java.io.IOException;
```
Comment (tylerhou) [inline src/main/java/com/google/sps/servlets/ReactServlet2.java:8]: `MainAppServlet`?

File: src/main/java/com/google/sps/servlets/ReactServlet2.java [hunk 11]
```diff
@@ -8,13 +8,13 @@
...
 import javax.servlet.ServletException;
 
-@WebServlet(urlPatterns={"/myFolders","CreateCard","CreateFolder", "ImageInterface", "YoutubeInterface", "InsideFolder"})
+@WebServlet(urlPatterns={"/myFolders","createCard","createFolder", "imageInterface", "youtubeInterface", "insideFolder"})
```
Comment (tylerhou) [inline src/main/java/com/google/sps/servlets/ReactServlet2.java:11]: Shouldn't these be capitalized? (e.g. `InsideFolder` to match the React routes)

**Pull request merged**